specs/data_structures.md

```diff
@@ -232,6 +225,13 @@ Putting it all together, the encoding for signatures is:
 
 This encoding scheme is derived from [EIP 2098: Compact Signature Representation](https://eips.ethereum.org/EIPS/eip-2098).
 
+### PublicKey
+
+| name | type       | description              |
+| ---- | ---------- | ------------------------ |
+| `x`  | `byte[]` | `x` value of public key. |
+| `y`  | `byte[]` | `y` value of public key. |
```

Comment:
Fixed-size arrays should have their size in the spec (i.e. `byte[32]` here). Sure protobuf doesn't have fixed-size arrays, but other serialization schemes might, and we don't want the specs to depend on the internals or restrictions of the serialization scheme.